### DISCUSSION TITLE
Unknown vulnerabilities

### DISCUSSION THREAD

**Discussion by dkoci** [quay/clair#1457] [Q&A]
Hi.
All images have Unknown vulnerabilities (using quay-operator v3.6.2 with managed clair):

![image](https://user-images.githubusercontent.com/5565897/146065277-df095176-4b24-4a81-9302-70bf902b4e67.png)

![image](https://user-images.githubusercontent.com/5565897/146065349-8696dafd-079d-42bb-a14b-091bc206b412.png)

**Comment by hdonnay** [comment]
what distribution is used in the container?

**Reply by dkoci** [reply]
quay.io/projectquay/clair@sha256:8d3b84cefa549fe4ec10c7aba0f416b984522cc9328c06780d72a35a5bc1de05

**Reply by hdonnay** [reply]
You're using a clair image as the base of the image being analyzed? The Quay UI makes it look like it's some flavor of Debian, so I find that very unlikely.
